content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst

```diff
@@ -32,85 +31,172 @@ between locations in a company's inventory.
-   numerical value in the :guilabel:`Cost` field for products in the respective product category
-   creates a new record in the *Inventory Valuation* report to adjust the value of the product. The
-   :guilabel:`Cost` amount will then automatically update based on the average purchase price both
-   of inventory on hand and the costs accumulated from validated purchase orders.
+- :guilabel:`First In First Out (FIFO)`: tracks the costs of incoming and outgoing items in
+  real-time and uses the real price of the products to change the valuation. The oldest purchase
+  price is used as the cost for the next good sold until an entire lot of that product is sold. When
+  the next inventory lot moves up in the queue, an updated product cost is used based on the
+  valuation of that specific lot. This method is arguably the most accurate inventory valuation
+  method for a variety of reasons, however, it is highly sensitive to input data and human error.
+
+.. warning::
+   Changing the costing method greatly impacts inventory valuation. It is highly recommended to
+   consult an accountant first before making any adjustments here.
+
+.. seealso::
+   :doc:`using_inventory_valuation`
 
 When the :guilabel:`Costing Method` is changed, products already in stock that were using the
 :guilabel:`Standard` costing method **do not** change value; rather, the existing units keep their
 value, and any product moves from then on affect the average cost, and the cost of the product will
 change. If the value in the :guilabel:`Cost` field on a product form is changed manually, Odoo will
 generate a corresponding record in the *Inventory Valuation* report.
 
-On the same screen, the :guilabel:`Account Stock Properties` fields will appear, as they are now
-required fields given the change to automated inventory valuation. These accounts are defined as
-follows:
+.. _inventory/management/config-inventory-valuation:
+
+Inventory valuation
+-------------------
+
+Inventory valuation in Odoo can be set to be updated manually or automatically. While *Expense*
+accounts apply to both, the *Stock Input* and *Stock Output* accounts are only used for automated
+valuation.
+
+Refer to the :ref:`Expense <inventory/management/expense-account>` and :ref:`Stock input/output
+<inventory/management/stock-account>` sections for details on configuring each account type.
+
+.. _inventory/management/expense-account:
+
+Expense account
+~~~~~~~~~~~~~~~
+
+To configure the *expense account*, go to the :guilabel:`Account Properties` section of the intended
+product category (:menuselection:`Inventory app --> Configuration --> Product Categories`). Then,
+choose an existing account from the :guilabel:`Expense Account` drop-down menu.
+
+To ensure the chosen account is the correct :guilabel:`Type,` click the |external link| icon to the
+right of the account. Then, set the account type based on the information below.
+
+.. tabs::
+   .. group-tab:: Anglo-Saxon
```

Comment:
I really like how you used the group tabs here, I think it works well to layout this information.
Please just add an empty line above and below each directive :)
```suggestion

   .. group-tab:: Anglo-Saxon
```